zfs_test/replicate_test/compress_test/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't think you need this one anymore, tbh.

Reply:
Yep, I'll drop those in a follow up diff.